docs/guides/insar_product_guide_template.md

```diff
@@ -121,6 +123,8 @@ In some cases, atmospheric errors can be corrected by using an atmospheric model
 
 ***Always doubt your interferogram first!*** View the interferogram critically, and consider if fringe patterns could potentially be driven by atmospheric effects. In general, it is best to avoid drawing conclusions from the outcome of a single interferogram.
 
+Tropospheric phase may be less impactful when considering small-scale deformation. As such, if you are using ASF's [Sentinel-1 Burst InSAR products](burst_insar_product_guide.md "Sentinel-1 Burst InSAR Product Guide" ){target=_blank} to look at deformation signals that are smaller than 1 km&sup2; you should consider using methods other than the typical atmospheric model based corrections to remove the effects of atmospheric delay.
```

Comment:
What are these other methods? I was under the impression that tropospheric corrections might not be required for the burst InSAR products. If that's not the case, it would be helpful to point users to approaches that *are* suitable for smaller deformation signals.

Reply:
@hjkristenson I've updated this section with the approaches that I think are most reasonable. Let me know what you think!

Reply:
Looks great @forrestfwilliams. Thanks!